### ISSUE TITLE
Get test.html working

### ISSUE THREAD

**Issue by SirPL** [bbloomf/jgabc#10]
Hello,
First of all, I'd like to apologize for my english and say that I very appreciate work you've done. jgabc is almost exactly what I need. Almost, because I'm looking for a way of getting fully passive behavior of jgabc. More accurately: the way test.html was working in one of the first releases (i.e. a0ce9fda752f03c23d9b86c66949f8a02b0773fb). So rendering based on gabc source in `div`, no interactivity (no neumes color change on hover, no textbox to edit syllables, no playback).
Is it hard to get it working that way?
Thank you for help and/or responses.

**Comment by sethborders** [comment]
Ben can answer more definitively, but you may want to take a look into how
it is used in the compline project:
https://bbloomf.github.io/compline/

**Comment by bbloomf** [comment]
Yes, I was just going to mention that [compline](https://bbloomf.github.io/compline/) [project](https://github.com/bbloomf/compline), which uses it that way, and uses a more recent version than the release that you mentioned.  I think that may have been just before I added some of the interactivity, so you could just try using the version of jgabc.js from that project.  Though I probably should fix it so that it can work in fully passive situations as well, and I don't think it would be difficult to do that, but I haven't had much time for working on side projects lately.

Thanks,

Ben

**Comment by SirPL** [comment]
That's exactly what I needed. Thank you very much.

**Comment by SirPL** [comment]
Compiline project is not working as good as I thought. It doesn't work in all browsers except Firefox... So I'm patiently waiting for passive mode of jgabc.

**Comment by bbloomf** [comment]
OK, I think it is starting to work pretty well again.  Take a look, and let me know what you think.

**Comment by SirPL** [comment]
I can confirm, that ligatures are now working in Firefox... But when it comes to passive mode, I see no difference. Which files have been updated?

**Comment by bbloomf** [comment]
You should be able to see that from looking at the commit logs.  However, now I see that I only committed the latest changes to the gh-pages branch, so if you weren't simply trying https://bbloomf.github.io/jgabc/test.html then you might not have seen the changes I made that affect passive mode.  I will merge them into the master branch right now

**Comment by SirPL** [comment]
Oh, I though you had commited it to master... So yes, I can confirm that test is working properly on all browsers except IE10.

**Comment by bbloomf** [comment]
I just tried it in IE10, and it seemed to work, although it took quite a long time to render.  What happened for you?

**Comment by SirPL** [comment]
Sorry, I was wrong - Thats IE11, not 10. What happened? I can see only names of staff (Kyrie XVII: etc...). I checked svg, and all of them looks like this:
```
<svg style="width: 100%; height: 0px">
  <defs> ... </defs>    <!-- defs contains all definitions -->
  <g class="caeciliae" transform="translate(0 42)"></g>
</svg>
```
And that's it. I have also found bug. There's problem with space between the first neume (or bar) in the line and the clef:

![somethingwrong](https://cloud.githubusercontent.com/assets/11311649/13554714/ddf59756-e3af-11e5-9fe7-a6575359d79c.png)
